src/API/index.ts

```diff
@@ -1 +1,45 @@
-export * from './API';
+import { TaskEither, tryCatch } from 'fp-ts/lib/TaskEither';
+import { identity } from 'fp-ts/lib/function';
+import { Restaurant } from 'src/model';
+import axios, { AxiosRequestConfig, AxiosPromise } from 'axios';
+import { stringify } from 'qs';
+
+const makeRequest = (category: string, radius?: string, location?: string): AxiosPromise => {
+  {
+    const { apiEndpoint, token } = process.env;
```

Comment:
invece di process.env, prova a usare config per leggere questi parametri,
dovresti creare un file `config/development.conf` simile a questo:
```
{
  "port": 3001,
  "bundle": {
    "apiEndpoint": "http://localhost:1337/localhost:8081",
  }
}
```
che rispecchi la struttura data in Config.js, poi definire un `config.d.ts` module per poterlo importare

Reply:
@Cardu Updated